### PR TITLE
[env] Add test cases in flink-env test suite

### PR DESCRIPTION
**Pull request by masteryhx**
## What is the purpose of the change

Add test cases for flink env

## Brief change log

  -  Add test cases (testDirOperation/testFileOperation/testGetChildren/testFileReadAndWrite) in `env_flink_test_suite`

## Verifying this change

This change added tests in `env_flink_test_suite` and can be verified by `make run_flink_test`

### CODE REVIEW

File: env/flink/env_flink_test_suite.cc [hunk 1]
```diff
@@ -45,22 +55,118 @@ void EnvFlinkTestSuites::setUp() {
...
-void EnvFlinkTestSuites::testFileExist() {
-  std::string fileName("test-file");
-  Status result = flink_env_->FileExists(fileName);
-  ASSERT_TRUE(result.IsNotFound());
+void EnvFlinkTestSuites::testDirOperation() {
+  const std::string dir_name = "test-dir";
+  ASSERT_TRUE(flink_env_->FileExists(dir_name).IsNotFound());
+  ASSERT_TRUE(flink_env_->CreateDir(dir_name).ok());
+  ASSERT_TRUE(flink_env_->CreateDirIfMissing(dir_name).ok());
+  ASSERT_TRUE(!flink_env_->CreateDir(dir_name).ok());
```
Comment (ljz2051) [inline env/flink/env_flink_test_suite.cc]: nit: How about introducing the "ASSERT_FALSE" macro definition?
For example: 
`#define ASSERT_FALSE(condition) ASSERT_TRUE(!condition)`

**Review by ljz2051 — COMMENTED** [review]
Thanks for your PR!  Overall LGTM. I left one minor comment.

File: env/flink/env_flink_test_suite.cc (hunk 1 above)
Comment (masteryhx) [reply env/flink/env_flink_test_suite.cc]: Thanks for the suggestion.
I have modified accordingly.

**Review by ljz2051 — APPROVED** [review]
Thanks for the update! LGTM.

**Pull request merged**